From 46461012e3fb5a1672346abdff7c16ea2786ee93 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 10 Jan 2014 17:04:28 +0000
Subject: [PATCH] builder: List --install packages in the same order as on the
 command line.

Previously the list was reversed.  Note this doesn't have any effect
on how the packages are installed since the guest package manager
doesn't care about the order.

(cherry picked from commit 39bc959c5b3a972e9a12921e002f27aaab2cd5ff)
---
 builder/cmdline.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builder/cmdline.ml b/builder/cmdline.ml
index 99412bb..813fbd8 100644
--- a/builder/cmdline.ml
+++ b/builder/cmdline.ml
@@ -311,7 +311,7 @@ read the man page virt-builder(1).
   let format = match !format with "" -> None | s -> Some s in
   let gpg = !gpg in
   let hostname = !hostname in
-  let install = !install in
+  let install = List.rev !install in
   let list_long = !list_long in
   let memsize = !memsize in
   let mkdirs = List.rev !mkdirs in
-- 
1.8.4.2

